src/sdk/bucketspace_fs.js

```diff
@@ -378,10 +380,15 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
                         full_path: path.join(this.fs_root, namespace_bucket_config.write_resource.path) // includes write_resource.path + bucket name (s3 flow)
                     }, object_sdk);
                 } else if (namespace_bucket_config) {
+                    // S3 Delete for NSFS Manage buckets
                     const list = await ns.list_objects({ ...params, limit: 1 }, object_sdk);
                     if (list && list.objects && list.objects.length > 0) {
                         throw new RpcError('NOT_EMPTY', 'underlying directory has files in it');
                     }
+                    const config_data = await this.get_config_data(bucket_path);
```

Comment:
don't we have the _id in namespace_bucket_config._id?

Reply:
@romayalon `read_bucket_sdk_namespace_info()` return only namespace related info, such as `read_resources` and `write_resource`. I can get bucket details from `object_sdk.read_bucket_sdk_config_info()` method. I will replace this line of code with that method.